lib/simplecov/configuration.rb

```diff
@@ -255,9 +255,9 @@ def refuse_coverage_drop
     #   SimpleCov.add_filter 'app/models' # will reject all your models
     # * as a block which will be passed the source file in question and should either
     #   return a true or false value, depending on whether the file should be removed
-    #   SimpleCov.add_filter do |src_file|
-    #     File.basename(src_file.filename) == 'environment.rb'
-    #   end # Will exclude environment.rb files from the results
+    #     SimpleCov.add_filter do |src_file|
+    #       File.basename(src_file.filename) == 'environment.rb'
+    #     end # Will exclude environment.rb files from the results
```

Comment:
Yes. Before this, those two lines around `File.basename(src_file.filename) == 'environment.rb'` were considered paragraphs by Markdown parsers.